modules/apps/object/object-web/types/src/main/resources/META-INF/resources/js/components/ModalAddObjectDefinition.d.ts

```diff
@@ -18,7 +18,11 @@ interface IProps extends React.HTMLAttributes<HTMLElement> {
 	apiURL: string;
 	observer: Observer;
 	onClose: () => void;
-	storageTypes: string[];
+	storages: Storage[];
 }
+declare type Storage = {
```

Comment:
@igor-franca please, define the type before its usage. Thanks :)

Reply:
I sent a PR to Feliphe with this modification